go.mod

```diff
@@ -5,6 +5,7 @@ module github.com/openshift-kni/cnf-features-deploy
 // This should also be matched within this project at:
 //   - cnf-tests/Dockerfile*
 //   - openshift-ci/Dockerfile*
+//   - ztp/resource-generator/Containerfile
```

Comment:
I think adding this comment means I need a /approve for `infra` instead of `ztp` which is unfortunate

/cc @imiller0 @ffromani